tests_e2e/tests/bvts/vm_access.py

```diff
@@ -38,6 +38,9 @@
 
 class VmAccessBvt(AgentTest):
     def run(self):
+        if type(self._context.node.os).__name__ == 'CoreOs' and self._context.node.os.information.full_version.startswith('Flatcar'):
```

Comment:
Should we keep a list of skipped tests by distro? Or add a TODO here so we know to remove this when VMAccess becomes supported on Flatcar.

Reply:
good point, we already have 3 or so tests/test cases that we skip. i'll create work items for those